src/FreeDSx/Snmp/Message/EngineId.php

```diff
@@ -277,11 +264,9 @@ protected static function parseDataFormat(int $format, $data)
     }
 
     /**
-     * @param int $format
-     * @param $data
```

Comment:
Please keep these docblocks. In general I would prefer they be there since that is mostly the case elsewhere. But I understand that they are somewhat redundant now with these typehints.

Reply:
That's the point, why keep them when they just add cluter? Generally every decent CS requires them to be dropped, hm? They make sense only for iterables, mixed or when extending methods without typehints as those cases cannot be typehinted

Reply:
This is also a valid reason why they should not be kept https://github.com/FreeDSx/SNMP/blob/master/src/FreeDSx/Snmp/Request/TrapV1Request.php#L154

Reply:
PSR2 says nothing of docblocks. But I guess I'm starting to agree with you the more I look at the changes. Keeping them out is fine.

Reply:
OT: Personally, I really fancy this one https://github.com/doctrine/coding-standard. It's based on advanced sniffs from https://github.com/slevomat/coding-standard. Check it out :)